sonar-core/src/main/java/org/sonar/core/issue/DefaultIssueBuilder.java

```diff
@@ -114,9 +113,12 @@ public DefaultIssueBuilder effortToFix(@Nullable Double d) {
     return this;
   }
 
+  /**
+   * @deprecated since 5.5, manual issue feature has been dropped.
+   */
+  @Deprecated
   @Override
   public DefaultIssueBuilder reporter(@Nullable String s) {
```

Comment:
![INFO](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-info.png) Do not forget to remove this deprecated code someday. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://dory.sonarsource.com/coding_rules#rule_key=squid%3AS1133)